doc/developers-guide/SCRAM-serialization.md

```diff
@@ -0,0 +1,22 @@
+## Overview
+
+This document describes the SCRAM serialization format used by MongooseIM.
+Developers can use this information to create advanced endpoints for `ejabberd_auth_http` or enable other software to read (i.e. share) the user authentication data.
+
+## Format description
+
+`==SCRAM==,<stored key>,<server key>,<salt>,<iteration count>`
+
+* `<stored key>` - Base64-encoded Stored Key
+* `<server key>` - Base64-encoded Server Key
+* `<salt>` - Base64-encoded Salt
+* `<iteration count>` - Iteration Count formatted as a human-readable integer
+
+In order to learn more about the meaning of Stored Key, Server Key, Salt and Iteration Count, please check [the SCRAM specification](https://tools.ietf.org/html/rfc5802).
```

Comment:
In order to learn more about the meaning of the Stored Key, Server Key, Salt and Iteration Count, please check [the SCRAM specification](https://tools.ietf.org/html/rfc5802).